src/components/__module.ts

```diff
@@ -12,6 +12,11 @@ import { ModuleConfig } from '../types-internal/module-config';
  * @property {EditorModules} Editor - List of Editor modules
  */
 export default class Module {
+  /**
+   * Module memorizes HTMLElements
+   */
+  public nodes: any;
```

Comment:
then, you should remove `public nodes(){}` from the modules? But it this case, it would not be clear of which nodes the module has 🤔

For example, for now, BlockSettings specify own nodes like this:

<img width="434" alt="image" src="https://user-images.githubusercontent.com/3684889/83899588-61a39c00-a761-11ea-8385-75791159b48f.png">
— it is clear structure.


We need to think about it.

Reply:
I see, that you have already remove the `nodes` from the modules, except those which are included at other modules. I will think about it.